### PR TITLE
documentation for listfunds, newaddr and withdraw

### PR DESCRIPTION
**Pull request by fixone**
some of the opened items in #1360

### CODE REVIEW

**Comment by fixone** [comment]
modified as suggested

**Comment by ZmnSCPxj** [comment]
ACK 86ff44bc855280cb492286de7303b49e248b6cf1

File: doc/lightning-withdraw.7.txt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+DESCRIPTION
+-----------
+The *withdraw* RPC command sends funds from c-lightning's 
+internal wallet to the address specified in 'destination'.
+
+The address can be of any Bitcoin accepted type, 
+including bech32.
+
+'satoshi' is the amount to be withdrawn from the internal 
+wallet (expressed, as name suggests, in satoshi)
```
Comment (ZmnSCPxj) [inline doc/lightning-withdraw.7.txt]: `'satoshi' can be the string '"all"' meaning to withdraw all available funds to the specified address, and to subtract fees from those funds.`

File: doc/lightning-connect.7.txt [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+(peer's public key)
+
+ERRORS
+------
+If 'node_id' or 'ip_addr' are invalid or if 'hostname' 
+can't be resolved to a valid IP address an error message
+will be returned. An error will also be returned if the 
+simplified version (only 'node_id') is used and there's
+no published IP for the peer. *connect* will make up to 
+10(?) attempts to connect to the peer before giving up
```
Comment (ZmnSCPxj) [inline doc/lightning-connect.7.txt]: #1322 will change this to a single attempt only.  Instead, persistent reconnecting (with exponential backoff until we do a retry every 5 minutes) will be done if we have a channel to that node (although this particular fact might be better mentioned in `fundchannel` manpage).

I suggest moving the `connect` manpage to a new pull request, as #1322 will change its behavior, so we can rewrite it separately after #1322 is merged in.

**Comment by ZmnSCPxj** [comment]
Commit title `mention 'add'` is incorrect, maybe `mention 'all'`

File: doc/lightning-connect.7.txt (hunk 2 above)
Comment (fixone) [reply doc/lightning-connect.7.txt]: Agree, removed commit ```0172d2```, will move it to a different PR

**Comment by ZmnSCPxj** [comment]
You removed `lightning-connect.7*`, but did not remove it from `doc/Makefile`.  I would suggest making a new branch `connect-manpage` from `ElementsProject/lightning/master`, cherry-picking the connect manpage commit onto it, then going back to `fixone/lightning/master` and rebase-interactive (`rebase -i`) and removing that commit from the rebase list.

**Comment by fixone** [comment]
Yes, noticed error from Travis, pushed Makefile again

**Comment by fixone** [comment]
The CI build failure doesn't seem anymore related to these changes, right?

File: doc/lightning-listfunds.7.txt [hunk 3]
```diff
@@ -0,0 +1,63 @@
+LIGHTNING-LISTFUNDS(7)
+======================
+:doctype: manpage
+
+NAME
+----
+lightning-listfunds - Protocol showing all funds currently managed by
```
Comment (cdecker) [inline doc/lightning-listfunds.7.txt]: This is a command, not a protocol.

File: doc/lightning-newaddr.7.txt [hunk 4]
```diff
@@ -0,0 +1,47 @@
+LIGHTNING-NEWADDR(7)
+====================
+:doctype: manpage
+
+NAME
+----
+lightning-newaddr - Protocol for generating a new address to
```
Comment (cdecker) [inline doc/lightning-newaddr.7.txt]: Same here

File: doc/lightning-withdraw.7.txt [hunk 5]
```diff
@@ -0,0 +1,52 @@
+LIGHTNING-WITHDRAW(7)
+====================
+:doctype: manpage
+
+NAME
+----
+lightning-withdraw - Protocol for withdrawing funds from the
```
Comment (cdecker) [inline doc/lightning-withdraw.7.txt]: And here :wink:

File: doc/lightning-listfunds.7.txt (hunk 3 above)
Comment (ZmnSCPxj) [reply doc/lightning-listfunds.7.txt]: Existing manpages use this term, but I am uncertain, how to differentiate "command" and "protocol".

Comment (cdecker) [reply doc/lightning-listfunds.7.txt]: Must have missed that until now, sorry. Being an RPC we can call it procedure, but protocol is definitely wrong.

Comment (fixone) [reply doc/lightning-listfunds.7.txt]: If it helps I also pushed ```dd06cdd``` with the same replacement in the rest of the pages

**Comment by cdecker** [comment]
Great, thanks @fixone, could you rebase and squash this? No need to keep the intermediate changes.

**Comment by fixone** [comment]
Would like to rebase, but I'm not sufficiently skilled on this topic to be able to get around this conflict on lightning-invoice.7 which is derailing my actions. Would appreciate some help.

**Comment by fixone** [comment]
Squashed the commits together, hopefully it's ok

**Comment by fixone** [comment]
This has become way to complicated (a mess mostly of my own making) & I think that no-one will have objections if it close this and open a much simpler PR with just a couple of commits.

**Pull request closed without merging**